sysutils/fontmatrix/Portfile

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+if {${os.platform} eq "darwin" && ${os.major} < 11} {
+    # Until this is fixed: https://github.com/fontmatrix/fontmatrix/issues/40
+    PortGroup       qt4 1.0
+
+    github.setup    fontmatrix fontmatrix 55f688aafb7c07899fade13dd8d8ec8663127e35
+    version         0.9.99
+    revision        0
+    checksums       rmd160  9f4b52ebce86ca35bd1e442c773c79b758e0e388 \
+                    sha256  5c10a0e466de71c85f652005d2ba1a5b3b898c47b37528d9f7364fcc2539d2a7 \
+                    size    2110252
+} else {
+    PortGroup       qt5 1.0
+
+    github.setup    fontmatrix fontmatrix 082f9379ad33987619addd790f473030da6deabe
+    version         0.9.100
```

Comment:
there is a version tagged upstream s `0.9.100` and thus that should be used instead of taking a git commit hash

Reply:
There is, but dating back to 2020, while there are several commits since then, including Qt-related ones. I do not really see any point in using a tarball with several patches applied (and identical result): it just adds unnecessary complication. And it is common for small projects not to have regular tarballs released.

Reply:
I don't agree - there are just 9 commits and the only "substantial" commit is fixing a deprecation warning for Qt5. So no, please just use the released version as there is no benefit to any of these additional commits. 

Also, the livecheck will work just fine as it always does ;)

Reply:
Ok, whatever, I will change it to use released ver.